.rubocop.yml

```diff
@@ -2,7 +2,6 @@ require: rubocop-rails
 
 AllCops:
   NewCops: enable
-  TargetRubyVersion: 2.6
```

Comment:
Not required when you have a ruby-version file, and can mean checks get out of sync.